### ISSUE TITLE
Part 4 troubleshooting otphelper

### ISSUE THREAD

**Issue by noobpawn10** [Plailect/Guide#300]
Validation stage 1 and 2 failed. According to the guide i must restore sysnand_original.bin but after the process the restoration failed saying that the file is corrupt

**Comment by noobpawn10** [comment]
EDIT: the troubleshooting guide also said to restore emunand_formatted.bin and it succeeded. So my only problem is still the sysnand_original.bin. I check and recheck and my sysnand backup is 1.84gb or 1.9m bytes. I was hoping for a quick response from anyone since this part of the guide says not to do anything or MY UNIT MAY BRICK. thanks

**Comment by Plailect** [comment]
Use your part 3 sysNAND.bin instead

**Comment by noobpawn10** [comment]
Sir Plailect, i restored using sysNAND.bin and it still fails saying that the file is corrupt. I tried doing one click setup but then validation stage 0,1and 2 failed. I then injected sysNAND using firm.bin and firm0 and firm1 were successful. Then I tried restoring again but now sysNAND, sysNAND_original, and emunand_formatted.bin fails and is corrupt. I have checked each one and they are properly sized. Is there any other workaround for this? Also, if your next workaround   still fail, is there a way to restore my unit back to factory state and start this guide all over again? Thanks
